pymc3/sampling.py

```diff
@@ -334,8 +334,28 @@ def sample(
     Notes
     -----
     Optional keyword arguments can be passed to ``sample`` to be delivered to the
-    ``step_method``s used during sampling. In particular, the NUTS step method accepts
-    a number of arguments. Common options are:
+    ``step_method``s used during sampling.
+
+    If you let ``sample()`` automatically assign the ``step_method``(s), and you
+    can correctly anticipate what they will be, then you can address them here
+    with keys that are (usually) the snakey_lowercase names of the step method.
+    e.g. to address:
+        1. ``target_accept`` to NUTS: nuts={'target_accept':0.9}
```

Comment:
for nuts this also works if you pass it directly, right? This might make it sound like you always have to use this extra-nesting.

Reply:
Yes, it works if you pass it directly, i.e `pm.sample(target_accept=0.9)`. You have to pass the step method only when the sampler uses several step methods.
So, I agree with Thomas and think this clarification should be mentionned.

Reply:
Thanks for the feedback guys - I've clarified the language in my latest commit